spec/models/institution_builder_spec.rb

```diff
@@ -933,17 +934,6 @@ def check_ipeds_ic_ay
       end
     end
 
-    describe 'when setting count_of_caution_flags' do
```

Comment:
Check vets-website didn't see a usage of this, there is already `institutions.caution_flag` that is a boolean that can be used as well